integration_tests/run_pyspark_from_build.sh

```diff
@@ -376,11 +376,9 @@ EOF
         exit 1
       fi
       export PYSP_TEST_spark_jars="${PYSP_TEST_spark_jars},${HYBRID_BACKEND_JARS//:/,}"
-      export PYSP_TEST_spark_rapids_sql_parquet_useHybridReader=true
```

Comment:
By-purpose to exclude this?

Reply:
Yes, cuz this config is not a start-only config.